Blizzard/node-rdkafka
# Feat consume topic per partition

How to test:

- clone branch locally
- run kafka with a broker on localhost:9093
- run `node ./test-create-topic-with-messages.js` to create a topic with 2 partitions with each 500 messages in it. Verify via Redpanda that the topic has been created. Kill the command manually.
- run `node ./test-consume.js` to start the consumer. This should consume the 2 partitions at a different speed. 

When testing multiple times, make sure to update the group.id so that the consumer group offset is reset.
